mods/env_sounds/depends.txt

```diff
@@ -0,0 +1 @@
+default
```

Comment:
`depends.txt` is deprecated, `mod.conf` should be used instead.

Reply:
Yes =)

Reply:
🙂